next/pages/guide/product/breakpoints.tsx

```diff
@@ -0,0 +1,141 @@
+                designs, each layout should be built to represent what the expected layout should be
+                between the breakpoints, not at the breakpoint.
+            </P>
+            <P>
+                <strong>Note:</strong> In rare cases a layout will need to change at a breakpoint
+                other than three provided. Add them with caution and provide comments explaining why
+                they are needed.
+            </P>
+
+            <Table>
+                <thead>
+                    <tr>
+                        <TH>Size</TH>
+                        <TH>Breakpoint</TH>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <TD>Small</TD>
+                        <TD>
+                            <Code>375px</Code>
+                        </TD>
+                    </tr>
+                    <tr>
+                        <TD>Medium</TD>
+                        <TD>
+                            <Code>600px</Code>
+                        </TD>
+                    </tr>
+                    <tr>
+                        <TD>Large</TD>
+                        <TD>
+                            <Code>946px</Code>
+                        </TD>
+                    </tr>
+                </tbody>
+            </Table>
+
+            <ThemedLink
+                theme="primary"
+                to="https://www.figma.com/file/ILP0YIrg8YsGJLeQChrZpsRi/Thumbprint-Web?node-id=1441%3A120"
+            >
+                View Figma resource
+            </ThemedLink>
+
+            <H2>Implementing breakpoints in code</H2>
+            <P>
+                Thumbprint provides Sass mixins that produce the needed code. In each of these
+                examples we are using breakpoint variables, however custom pixel widths are valid if
+                needed. Because we are &quot;small-first&quot; (aka &quot;mobile-first&quot;),
+                styles will often be grouped as follows:
+            </P>
+            <Table>
+                <thead>
+                    <tr>
+                        <TH>Size</TH>
+                        <TH>Breakpoint</TH>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <TD>Small</TD>
+                        <TD>
+                            <Code>481px</Code>
+                        </TD>
+                    </tr>
+                    <tr>
+                        <TD>Medium</TD>
+                        <TD>
+                            <Code>700px</Code>
+                        </TD>
+                    </tr>
+                    <tr>
+                        <TD>Large</TD>
+                        <TD>
+                            <Code>1025px</Code>
+                        </TD>
+                    </tr>
+                </tbody>
+            </Table>
+
+            <Code language="scss">
+                {`
```

Comment:
Is this how Prettier formats this? I feel like the indentation could be clearer. nbd if you're replacing it with a new code component though.

Reply:
Yeah, IIRC Prettier is a bit weird about the formatting.